recipe/meta.yaml

```diff
@@ -52,8 +52,8 @@ test:
 
 about:
   home: https://software.ecmwf.int/wiki/display/ECC/ecCodes+Home
-  license: Apache-2.0
-  license_file: ['LICENSE']
+  license: Apache 2.0
```

Comment:
We try to use the SPDX short identifier as much as possible, so this should be Apache-2.0. See https://spdx.org/licenses/Apache-2.0.html